pom.xml

```diff
@@ -13,6 +13,15 @@
         <maven.compiler.source>11</maven.compiler.source>
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <!-- Plugins -->
+        <version.liberty-maven-plugin>3.11.1</version.liberty-maven-plugin>
+        <!-- OpenLiberty runtime -->
+        <version.openliberty-group>io.openliberty</version.openliberty-group>
+        <version.openliberty-runtime>RELEASE</version.openliberty-runtime>
+        <http.port>9080</http.port>
+        <https.port>9443</https.port>
+        <liberty.var.app.context.root>/</liberty.var.app.context.root>
```

Comment:
These
```suggestion
        <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
        <!-- Plugins -->
        <version.liberty-maven-plugin>3.11.1</version.liberty-maven-plugin>
        <!-- OpenLiberty runtime -->
        <version.openliberty-group>io.openliberty</version.openliberty-group>
        <version.openliberty-runtime>RELEASE</version.openliberty-runtime>
        <http.port>9080</http.port>
        <https.port>9443</https.port>
        <liberty.var.app.context.root>/</liberty.var.app.context.root>
```
These are provided below, if you'd prefer to keep all of the properties here then you'll need to change the values below to variables